ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -343,7 +346,7 @@ private IQueryable<Notification> GetBannerReadyNotificationsIQueryable()
                 .Include(n => n.SocialContextVenues)
                 .Include(n => n.TreatmentEvents)
                     .ThenInclude(t => t.TreatmentOutcome)
-                .AsSplitQuery();
+                .OrderBy(n => n.NotificationId);
```

Comment:
Why have we got rid of the `AsSplitQuery` here?

Reply:
I have moved it to the base queryable that this is using, as that is also used by other functions, so we don't have to add it to every function that uses the base queryable.